test/testlib/nuodb_admin_utilities.go

```diff
@@ -177,3 +200,26 @@ func AwaitNrLoadBalancerPolicies(t *testing.T, namespace string, podName string,
 		return err == nil && len(policies) == expectedNumber
 	}, 30*time.Second)
 }
+
+func ApplyNuoDBLicense(t *testing.T, namespace string, adminPod string) {
+	options := k8s.NewKubectlOptions("", "", namespace)
+	licenseContent := os.Getenv("NUODB_LICENSE_CONTENT")
+	if licenseContent != "" {
+		licenseContentBytes, err := base64.StdEncoding.DecodeString(licenseContent)
```

Comment:
Is there any risk of someone creating a fork of this repo and making a change to print out the license, so that they can use it they can copy the license text and use it in their own system? I think they would have to publish a PR so that tests are run on our CircleCI, so we would at least notice it.

Reply:
This is a good question!
According to [Circle CI doc](https://circleci.com/docs/2.0/oss/#build-pull-requests-from-forked-repositories):

> Running an unrestricted build in a parent repository can be dangerous. Projects often contain sensitive information, and this information is freely available to anyone who can push code that triggers a build.
By default, CircleCI does not pass secrets to builds from forked PRs for open source projects and hides four types of configuration data: ...

I decided to test this explicitly, however, it seems that Circle CI builds are disabled currently on Forked Pull requests.
See https://github.com/nuodb/nuodb-helm-charts/pull/191.
@vegichan If you decide to enable this feature, we could double-check that configured secrets are not leaked in the above PR.

More on this topic:
* https://circleci.com/blog/managing-secrets-when-you-have-pull-requests-from-outside-contributors
* https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/